package.json

```diff
@@ -28,15 +28,15 @@
   "license": "MIT",
   "dependencies": {
     "@microsoft/load-themed-styles": "^1.7.59",
-    "office-ui-fabric-react": "~5.112.0",
-    "react": "~15.3.2",
-    "react-dom": "~15.3.2",
+    "office-ui-fabric-react": "^7.53.0",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
     "roosterjs-editor-plugins": "^7.2.5",
     "roosterjs-plugin-image-resize": "^7.2.5"
```

Comment:
Do roosterjs-editor-plugins and roosterjs-plugin-image-resize versions not need to be updated?

Reply:
neither of them take a dependency on react, so they can be bumped as part of a separate change.

I bumped `oufr`, because oufr only supports react 16 after oufr 7.x